src/lucky_web/support/int32.cr

```diff
@@ -0,0 +1,5 @@
+require "../allowed_in_tags"
+
+struct Int32
```

Comment:
This should go in `src/charms/*.cr` That's where all the extensions to build in types go